core/src/main/java/com/orientechnologies/orient/core/Orient.java

```diff
@@ -271,24 +271,8 @@ public Orient startup() {
       final int cores = Runtime.getRuntime().availableProcessors();
 
       workers =
-          new OThreadPoolExecutorWithLogging(
```

Comment:
Will the new implementation be blocking ? looking at I think is not, would be better that this executor is kept blocking, may create some behaviour problem changing it to not blocking.

Reply:
That was one of the questions I had - I've added blocking behaviour back in now.